Content.Client/Construction/UI/FlatpackCreatorMenu.xaml.cs

```diff
@@ -100,6 +106,17 @@ protected override void FrameUpdate(FrameEventArgs args)
                 (_currentBoard.Value, machineBoard));
             CostLabel.SetMarkup(GetCostString(cost));
         }
+        else if (_currentBoard != null &&
+                 _entityManager.TryGetComponent<ComputerBoardComponent>(_currentBoard, out var computerBoard) &&
+                 computerBoard.Prototype != null)
+        {
+            var proto = _prototypeManager.Index<EntityPrototype>(computerBoard.Prototype);
+            _machinePreview = _entityManager.Spawn(proto.ID);
+            _spriteSystem.ForceUpdate(_machinePreview.Value);
+            MachineNameLabel.SetMessage(proto.Name);
+            var cost = _flatpack.GetFlatpackCreationCostForComputer((_owner, flatpacker));
+            CostLabel.SetMarkup(GetCostString(cost));
+        }
```

Comment:
same deal here. Just have a small split for getting the prototype and then don't duplicate any more code.